internal/cmd/functional-benchmarker/config/config.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+const (
+	FluentdBaselineConf = `
+<system>
+  log_level debug
+</system>
+
+<source>
+  @type tail
+  @id container-input
+  path "/var/log/containers/*.log"
+  pos_file "/var/lib/containers.log.pos"
+  refresh_interval 5
+  rotate_wait 5
+  tag kubernetes.*
+  read_from_head "true"
+  @label @FLUENTDFORWARD
+  <parse>
+    @type regexp
+    expression /^(?<@timestamp>[^\s]+) (?<stream>stdout|stderr)( (?<logtag>.))? (?<log>.*)$/
+    time_format '%Y-%m-%dT%H:%M:%S.%N%:z'
+    keep_time_key true
+  </parse>
+</source>
```

Comment:
I don't think we can avoid the concat stage, for post-CRI-O reassembly

Reply:
this is a baseline config currently unused